action.yaml

```diff
@@ -137,6 +137,8 @@ runs:
       uses: docker/build-push-action@v2
       with:
         secrets: ${{ inputs.build-secrets }}
+        context: ${{ inputs.docker-context }}
```

Comment:
If there's a default in the docs, there should be a default in the action, and you need to define these inputs in the inputs

Reply:
Yup, you are correct, added the inputs =cP

I will add a default to docker-file, because we don't need to delegate the default to the docker build action and possibly get out of sync with the docs, unlikely as it is.

The default I put in the readme for docker-context is from the docker build action, the default for the context input. Like the above we could make a separate default, or we can update the docs to be more direct that it's a default from a sub action. What do you recommend?

Reply:
If they have a default in the sub action, our default should match it, otherwise we're overriding their default because an empty string is a value.

Reply:
For anyone else who comes along, after some discussion it was found that because we use the `actions/checkout` action we can set the default of `docker-context` to `./` with no need to worry about Git Context. Documentation regarding git context found [here at this commit, first paragraph below the codeblock](https://github.com/docker/build-push-action/tree/42863b1282f7046f6c1ec9222965bbceafae3bb8#git-context)